Gemfile.lock

```diff
@@ -360,25 +354,26 @@ GEM
     sync (0.5.0)
     sysexits (1.2.0)
     temple (0.8.2)
-    thin (1.7.2)
+    thin (1.8.1)
```

Comment:
Absolutely @NickLaMuro :point_up:

Reply:
Oh!  You did a generic `bundle update`, not just for celluloid.  Cool beans.  Thanks!

Reply:
Yeah celluloid was the only one which was failing on the latest version so I limited it to the prior version.

Reply:
celluloid was all the rage for a while but died out after sidekick rewrote to not include it.
TBH, kinda surprised to see it.